apps/backend/src/libs/modules/controller/base-controller.module.ts

```diff
@@ -29,9 +30,30 @@ class BaseController implements Controller {
 		this.logger.info(`${request.method.toUpperCase()} on ${request.url}`);
 
 		const handlerOptions = this.mapRequest(request);
-		const { payload, status } = await handler(handlerOptions);
+		const handlerResponse = await handler(handlerOptions);
```

Comment:
preHandler should be added to a specific addRoute, no need for mapPreHandler and other logic in this file